packages/main/src/Form.ts

```diff
@@ -367,6 +368,30 @@ class Form extends UI5Element {
 
 	get groupItemsInfo(): Array<GroupItemsInfo> {
 		return this.items.map((groupItem: IFormItem) => {
+			const items = this.getItemsInfo((Array.from(groupItem.children) as Array<IFormItem>));
+			const breakpoints = ["S", "M", "L", "Xl"];
```

Comment:
extract breakpoints - no need to declare it for every loop and re-rendering all the time.